eos-tech-support/eos-stage-flatpak

```diff
@@ -57,7 +57,7 @@ else
     flatpak_repo=/var/lib/flatpak/repo
 fi
 
-for remote in `flatpak remote-list | awk {'print $1'}` ; do
+for remote in `flatpak remote-list --system | awk {'print $1'}` ; do
```

Comment:
Sounds good.  The awk might not strictly be needed anymore, but I suppose it doesn't hurt.